lib/presentation/bottom_nav_screens/profile_screen/review_screen/review_screen.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:capi_restro/core/core.dart';
+import 'package:capi_restro/presentation/bottom_nav_screens/profile_screen/review_screen/review_items.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:go_router/go_router.dart';
+import 'package:percent_indicator/linear_percent_indicator.dart';
+
+class ReviewScreen extends StatelessWidget {
+  const ReviewScreen({
+    super.key,
+    required this.followers,
+    required this.following,
+    required this.reviews,
+    required this.comment,
+    required this.photos,
+    required this.foodietype,
+  });
+
+  final String followers;
```

Comment:
Instead of adding individual parameters create a data class and pass parameter as class properties